color-picker-project/src/components/hero/Hero.css

```diff
@@ -0,0 +1,90 @@
+.section-container {
+  position: relative;
+  display: grid;
+  min-height: 100vh;
+  place-content: center;
+  overflow: hidden;
+  background-color: var(--background-color);
+  padding: 24px;
+  text-align: center;
+  color: var(--text-color-light);
+}
+
+.content-container {
+  position: relative;
+  z-index: 10;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header-span {
+  margin-bottom: 6px;
+  display: inline-block;
+  background-color: rgba(75, 85, 99, 0.5);
+  padding: 6px 12px;
+  border-radius: 99px;
```

Comment:
Consider changing border radius to a percent instead of px?  Worth testing browser settings at increased or decreased root font sizes to see if it changes how the border radius looks.

Reply:
Okay, on it